src/controllers/update_events.js

```diff
@@ -0,0 +1,32 @@
+const query = require('../database/queries/update_events_query');
```

Comment:
Name your query something more readable. Like, what is it responsible for? and then ask yourself, what does this query do? What is it's input and what's the output? Your name should be very specific, and your input should be very relevant, convince me why is your query taking `{ params: req.params, body: req.body }` not just `{ id }`.